packages/credential-w3c/src/action-handler.ts

```diff
@@ -75,127 +52,30 @@ export class CredentialPlugin implements IAgentPlugin {
       },
     },
   }
+  private issuers: AbstractCredentialProvider[]
 
-  constructor() {
+  constructor({ issuers = [] }: { issuers: AbstractCredentialProvider[] }) {
```

Comment:
I think we should simplify the constructor to reduce nesting of parameters
```suggestion
  constructor( issuers: AbstractCredentialProvider[] ) {
```

Reply:
changed it to exactly how it's done in the message handlers